stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -349,4 +349,31 @@ public void onCancel() {
     public void onComplete() {
     }
   }
+
+  public static void unimplementedStreamResponse(StreamObserver responseObserver) {
```

Comment:
@zhangkun83, do you think we should go ahead and make one of these for each type of call: unary, serverStreaming, clientStreaming, bidiStreaming. (like we do for asyncUnaryCall, asyncServerStreamingCall, etc.)